dtmi/azureiot/phone_as_a_device/sensors/battery-1.json

```diff
@@ -0,0 +1,23 @@
+{
+    "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:azureiot:Phone_As_A_Device:Sensors:Battery;1",
+    "@type": "Interface",
+    "contents": [
+      {
+        "@type": "Property",
+        "displayName": {
+          "en": "Enabled"
+        },
+        "name": "enabled",
+        "schema": "boolean",
+        "writable": true
+      },
+      {
+        "@type": "Telemetry",
```

Comment:
semantic type?

Reply:
There's a semantic type for Percentage? https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md#semantic-types